services/admin/src/server.ts

```diff
@@ -176,26 +242,41 @@ export function buildApp({ store }: { store: Store }): Application {
     }
   );
 
-  app.get<NoParams>(
-    '/admin/write-ins/adjudications/contestId/count',
+  app.get<ElectionsNamespaceParams>(
+    '/admin/elections/:electionId/adjudications/grouped-by/contest-id',
     (_, response) => {
       response.json(store.getAdjudicationCountsGroupedByContestId());
     }
   );
 
-  app.get<NoParams>('/admin/write-ins/adjudications/counts', (_, response) => {
-    response.json(store.getAdjudicationCountsByContestIdAndTranscribedValue());
-  });
+  app.get<ElectionsNamespaceParams>(
+    '/admin/elections/:electionId/adjudications/grouped-by/contest-id-and-transcribed-value',
```

Comment:
Nit: I would expect something like `grouped-by` to be a query parameter in a normal REST API

Reply:
Yeah, I wasn't sure about how I wanted to handle this. I think these API endpoints probably won't survive long anyway.

Reply:
I ended up removing these API endpoints altogether.